### ISSUE TITLE
Select Calendars

### ISSUE THREAD

**Issue by nicmart-dev** [nicmart-dev/linguistnow#18]
Display the list of available calendars to the user and allow them to select the ones they are interested in, using the access token from #17.

**Comment by nicmart-dev** [comment]
```jsx
const listCalendars = async (accessToken) => {
  const calendar = google.calendar({ version: 'v3', auth: accessToken });

  const calendarList = await calendar.calendarList.list();
  return calendarList.data.items;
};
```

**Comment by nicmart-dev** [comment]
Alternative possible solution, still not involving n8n, where user can log in to select their calendars: https://blockchain.oodles.io/dev-blog/integrating-google-calendar-api-into-react-application/

**Comment by nicmart-dev** [comment]
Merged with #17
